test/models/AwrsModelSpec.scala

```diff
@@ -255,8 +255,8 @@ class AwrsModelSpec extends UnitSpec with AwrsTestJson {
       val awrsModel = Json.parse(finalDeletedJson).as[AWRSFEModel]
       val etmpJson = Json.toJson(awrsModel)(AWRSFEModel.etmpWriter).toString()
 
-      etmpJson should include("\"businessDetails\":{\"nonProprietor\":{\"tradingName\":\"asddsad\",\"identification\":{\"doYouHaveVRN\":false")
-      etmpJson should not include "\"businessDetails\":{\"nonProprietor\":{\"tradingName\":\"asddsad\",\"identification\":{\"doYouHaveVRN\":false,\"vrn\":\"000000000\""
+      etmpJson should include(s""""nonProprietor":{"tradingName":"asddsad","identification":{"utr":"${AwrsTestJson.testUtr}","doYouHaveVRN":false,"doYouHaveUTR":true""")
```

Comment:
I'm a bit unclear as to why 'businessDetails' is no longer part of the Json without the model changing. (Or unless it's a change that I haven't yet scrolled down to.)